book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rental
```

Comment:
Same here, Kindly make sure that there's a method for adding rentals similar to the example I demonstrated above for the `Person` class.

 i.e
```
  def add_rental(person, date)
    //the book itself should be sent as a parameter to create the new rental achieving the 'has-many' association. 
  end
```